client/src/components/Sle/Sle.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import "./Sle.css";
+import "./Sidebar.css";
+import "./Sidebar.jsx";
+import Sidebar from "./Sidebar";
+
+
+
+function Sle(props) {
+    return (
+    
+    <div>
+    
+    <div class = "line">
+    </div>
+    
+    <div class = "sidebar">
+      <Sidebar
+          title= "Your Shifts"/>
+      <Sidebar
+          title= "Open Shifts"/>
+      <Sidebar
+          title= "Availability"/>
+    </div>
+
+    <div class = "top-right">
```

Comment:
Let's make this top-right thing its own component because we're going to have to use it on every page.